apps/condo/domains/property/components/panels/BuilderPanel.tsx

```diff
@@ -0,0 +1,47 @@
+import { useIntl } from '@core/next/intl'
```

Comment:
What is `panels`?
Add readme with description in components/panels/

Reply:
Done